specification/recoveryservices/resource-manager/readme.md

```diff
@@ -43,6 +43,17 @@ message-format: json
 ```
 
 
+### Tag: package-preview-2022-10
```

Comment:
Confirmed with @jianyexi , the linter error can be resolved by adding suppression: https://github.com/Azure/azure-rest-api-specs/pull/2229/files#diff-c52cea7cbbcf550e62037de87a237d27R50

Reply:
Thanks, I've added suppression for that